com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -549,6 +549,7 @@ public override void OnDestroy()
         private ClientRpcParams m_ClientRpcParams;
         private AnimationMessage m_AnimationMessage;
 
+#if UNITY_EDITOR
```

Comment:
I understand the `#if` in `NetworkAnimatorTests.cs`. But why do need this one too? Couldn't we leave this expose, always, but not called outside the editor ?

Reply:
We could, but why keep an internal method for runtime builds that will never be used?

Reply:
Whether it is called or not is a consideration of the calling code. If we try to #if out everything that can't be code in any specific case, there will be an explosion of #if s all over the code. It's an internal method, it's there. All good. What I worry about is the #if that can lead to highly unmaintainable code

Reply:
Don't get me started on the time that [large Montreal game company] threw in a `#ifdef fifty_journalists` because it had a journalist delegation coming and needed a demo. It was supposed to be just for a build, for one afternoon. That define stayed there 6+ months, across multiples games and nobody seemed able to remove it once it started affecting the whole codebase.